script/jenkins/windows-build.ps1

```diff
@@ -43,6 +43,12 @@ New-Item -ItemType directory -Path build\coverage | Out-Null
 New-Item -ItemType directory -Path build\system-tests | Out-Null
 New-Item -ItemType directory -Path build\system-tests\run | Out-Null
 
+choco install python -y -r --no-progress --version 3.8.1.20200110
```

Comment:
why is this duplicated now?

Reply:
Just because I don't want to refactor this script too. The two scripts already are mostly overlapping - would be nice to clean that up, but I'd prefer to tackle it later.

Reply:
I think I missed the point of your question. It's needed here because the targets moved around a little bit. "make fields" now creates docs/fields.asciidoc, which requires the use of a Python script.